monai/handlers/checkpoint_saver.py

```diff
@@ -62,9 +62,9 @@ class CheckpointSaver:
             typically, it's used to resume training and compare current metric with previous N values.
         key_metric_greater_or_equal: if `True`, the latest equally scored model is stored. Otherwise,
             save the the first equally scored model. default to `False`.
-        key_metric_score_sign: sign of the score, available value: `[1.0, -1.0, 1, -1]`. For error-like metrics,
-            e.g. smaller is better, a negative score sign should be used (objects with larger score are retained).
-            default to `1.0`.
+        key_metric_negative_sign: whether adding a negative sign to the metric score to compare metrics,
```

Comment:
@Nic-Ma I think this change is from another PR ?

Reply:
Hi @vfdev-5 , yes, it's to address this comment: https://github.com/Project-MONAI/MONAI/pull/2487#discussion_r662082802.

Thanks.